backend/grant/utils/requests.py

```diff
@@ -21,9 +23,25 @@ def blockchain_get(path, params=None):
 
 
 def blockchain_post(path, data=None):
+    if E2E_TESTING:
```

Comment:
Ditto here. Looks like there are only backend changes, so it's pretty minimal. I'll let @wbobeirne jump in with his recommendation

Reply:
Same general feelings as above, though since they're coming up a _little_ more now here, might be nice to have something similar to mock's `@patch` that moves these logic shifts into an e2e specific file, rather than these libraries. Not sure what the python convention for this is.